docs/guides/package.md

```diff
@@ -177,6 +177,34 @@ to files in the registry, and it will also handle raced parallel uploads. Note t
 need to match exactly with those previously uploaded to the registry, this avoids accidentally
 publishing source distribution and wheels with different contents for the same version.
 
+### Uploading attestations with your package
+
+!!! note
+
+    Some third-party package indexes may not support attestations, and may
+    reject uploads that include them (rather than silently ignoring them).
+    If you encounter issues when uploading, you can use `--no-attestations` or
+    `UV_PUBLISH_NO_ATTESTATIONS` to disable uv's default behavior.
+
+!!! tip
+
+    `uv publish` does not currently generate attestations; attestations must
+    be created separately before publishing.
```

Comment:
Should we link or suggest how they would be created?

Reply:
The wrinkle here is that there's no _good_ way to do this currently 😅 -- we could point users to `pypi-attestations` as a DIY approach, but that's a glue package that isn't really intended for direct usage.

(What `gh-action-pypi-publish` does is have a helper script that uses the `pypi-attestations` APIs.)

One option here would be to create `astral-sh/pypi-attest` as an action, which we'd then recommend at least until we have in-client attesting. That would only take me an hour or two to build.